docs/api.md

```diff
@@ -475,6 +476,99 @@ Content:
 
 Students are identified by the `client_id` Cookie.
 
+
+### <a name="get-engagement"></a>Get Engagement
+
+Get a lectures engagement values
+
+#### URL
+
+`/lectures/:lecture-id/engagements`
+
+#### Method
+
+GET
+
+#### URL Parameters
+
+##### Optional
+
+* `last=[integer]` 1 or 0. 1 shows only last entry for each student.
+* `recent=[integer]` 1 or 0. 1 shows only recent entries (last 15 min)
```

Comment:
I think we can drop the recent parameter for now, it will make the code just a little bit simpler (less tests) and we don't need it.